routes/web.php

```diff
@@ -643,6 +643,9 @@ function (): void {
 Route::group(['prefix' => 'api'], function (): void {
     // Protected by a gate in the controller, instead of policy middleware.
     Route::get('jobs/{jobPoster}/assessment-plan', 'AssessmentPlanController@getForJob');
+    Route::get('users', 'Api\UserController@index');
+    Route::get('users/{user}', 'Api\UserController@show')
```

Comment:
Wouldn't I have to register it in `Kernel.php` with an associated Middleware file though? Right now it's a gate.

Reply:
I got the syntax wrong in my suggestion, it would be `->middleware('can:view-user', 'user')`. So you wouldn't have to register a new middleware, the 'can' middleware should work with any Gate or Policy.